proto/tenancy_config/ota_config.proto

```diff
@@ -0,0 +1,12 @@
+﻿syntax = "proto3";
+
+package holms.types.tenancy_config;
+option csharp_namespace = "HOLMS.Types.TenancyConfig";
+option go_package = "tenancyconfig";
+
+message OTAConfig {
+	string provider_i_d = 1;
+	string provider_user_name = 2;
+	string provider_password = 3;
+	string hotel_i_d = 4;
```

Comment:
@pcsouza 
should this be left as HotelID? I was following the object model you've created here:
https://github.com/shortbar/holms/blob/master/HOLMS.Application/Contexts/Booking/OTASync/CRCredentials.cs

Also wanted to make sure that despite the name you've used, it actually maps to the tenancy, and not the property.